iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergSerDe.java

```diff
@@ -218,7 +235,15 @@ private static Schema hiveSchemaOrThrow(Properties serDeProperties, Exception pr
       throws SerDeException {
     // Read the configuration parameters
     String columnNames = serDeProperties.getProperty(serdeConstants.LIST_COLUMNS);
+    // add partition columns to schema as well, if any
+    if (serDeProperties.getProperty(serdeConstants.LIST_PARTITION_COLUMNS) != null) {
+      columnNames = columnNames + "," + serDeProperties.getProperty(serdeConstants.LIST_PARTITION_COLUMNS);
+    }
```

Comment:
nit: newline

Reply:
Do you mean there should be a newline added after line 241?

Reply:
yeah